Gemfile.lock.release

```diff
@@ -1511,12 +1511,15 @@ DEPENDENCIES
   terminal
   time (~> 0.2.2)
   timecop (~> 0.9, != 0.9.7)
-  uri (>= 0.12.2)
+  uri (~> 0.13.1)
   vcr (~> 6.1)
   webmock (~> 3.7)
   websocket-driver (~> 0.6.3)
   wim_parser (~> 1.0)
   yard (>= 0.9.36)
 
+RUBY VERSION
+   ruby 3.1.6p260
```

Comment:
Shouldn't this be ruby 3.3?

Reply:
I was actually surprised this appeared at all, considering I'm using the same bundler version, but yeah I can update it.  It doesn't seem to make any difference.

Reply:
Updated